web/src/components/Plugin/PluginPage.tsx

```diff
@@ -180,16 +180,21 @@ const PluginPage: React.FC<Props> = ({
                   key={item.name}
                   actions={[
                     <Button
-                      type={
+                      type='primary'
+                      danger= {
                         initialData[item.name] && !initialData[item.name].disable
-                          ? 'primary'
-                          : 'default'
+                          ? true
+                          : false
                       }
                       onClick={() => {
                         setName(item.name);
                       }}
                     >
-                      Enable
+                      {
+                        initialData[item.name] && !initialData[item.name].disable
+                          ? 'Disable'
+                          : 'Enable'
+                      }
```

Comment:
Hi @juzhiyuan 

In here should I create the translation in locales file?

Reply:
ya